src/services/api-service.ts

```diff
@@ -310,8 +311,8 @@ class ApiService {
         process,
         chainId,
         blockHeight,
-        holographVersion: ApiService.holographVersion,
       },
+      holographVersion: ApiService.holographVersion,
```

Comment:
This is separate from the input now?

Reply:
Yep! It was a refactor on the API side, to make it more explicit and easier to integrate on the FE